server/schedule/operator_controller.go

```diff
@@ -410,6 +410,9 @@ func (oc *OperatorController) checkAddOperator(ops ...*operator.Operator) bool {
 			operatorWaitCounter.WithLabelValues(op.Desc(), "exceed-max").Inc()
 			return false
 		}
+		if fromAddmin || op.IsLeaveJointStateOperator() {
```

Comment:
prefer to use op.Kind()

Reply:
It seems the `LeaveJointStateOperator` share the same `kind ` with other operators.

Reply:
Oh, I mean `SchedulerKind`

Reply:
can you provide a further heavy hint? which `ScheduleKind` should it be when it is create by `leave joint state`? 😂

Reply:
There is no need to use `fromAddmin`, maybe you can use `op.SchedulerKind() == OpAdmin`